playground/frontend/lib/pages/embedded_playground/widgets/embedded_actions.dart

```diff
@@ -45,20 +45,13 @@ class EmbeddedActions extends StatelessWidget {
       children: [
         Consumer<PlaygroundController>(
           builder: (_, controller, __) {
-            final selectedExample = controller.selectedExample;
-            final hasGithubLink = selectedExample?.urlVcs != null;
-            final hasColabLink = selectedExample?.urlNotebook != null;
-
             return Row(
```

Comment:
The embedded playground can be narrow. Will all buttons always fit? Maybe GitHub and Colab icons should not have text in the embedded playground?

Reply:
done